ext/websocket/01_websocket.js

```diff
@@ -424,18 +432,23 @@
             this.dispatchEvent(event);
             break;
           }
-          case "ping": {
+          /* ping */
+          case 3: {
             core.opAsync("op_ws_send", this[_rid], {
               kind: "pong",
             });
```

Comment:
we can actually delete this because tungstenite auto-responds

Reply:
i don't want to touch funationality in this PR. maybe a follow up?

Reply:
sure